app/javascript/src/Plans/components/PlansTable.tsx

```diff
@@ -1,40 +1,120 @@
-import {
-  Button,
-  Divider,
-  Toolbar,
-  ToolbarContent,
-  ToolbarItem
-} from '@patternfly/react-core'
-import {
-  Table,
-  TableBody,
-  TableHeader,
-  sortable
-} from '@patternfly/react-table'
-
-import { Pagination } from 'Common/components/Pagination'
+import { useState } from 'react'
+import { Button, Divider, Pagination, Toolbar, ToolbarContent, ToolbarItem } from '@patternfly/react-core'
```

Comment:
Should Pagination be coming from 
```
import { Pagination } from 'Common/components/Pagination'
```
?

For me the pagination is broken (it's weird that on your screenshots it looks OK).

But that was fixed in https://github.com/3scale/porta/pull/3369/files

Reply:
Oops!!

Reply:
Done